contracts/IBorrowersRegistry.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+
+
+/**
+ * @title IBorrowersRegistry
+ * Keeps a registry of created trading accounts to verify their borrowing rights
+ */
+interface IBorrowersRegistry {
+
+  function canBorrow(address _account) external view returns(bool);
+
+  function getAccountForUser(address _user) external view returns(address);
```

Comment:
This is basically reverse function to the subsequent one, so maybe we could name it `getLoanForOwner`? Especially `Account` can be confusing for readers

Reply:
I agree, let's do it @piotruda

Reply:
done